builder/node.go

```diff
@@ -38,6 +39,8 @@ type Node struct {
 	Labels    map[string]string
 }
 
+const defaultDriverTimeout = 120 * time.Second
```

Comment:
as such:
- CLI by default has 20 seconds timeout, it can be changed by a user by providing CLI option or env. variable
- buildx driver factory has 120 seconds timeout, it cannot be changed by a user, it's up to a buildx driver developer to define that value